openmetadata-ui/src/main/resources/ui/src/rest/userAPI.ts

```diff
@@ -233,3 +234,40 @@ export const createUserWithPut = async (userDetails: CreateUser) => {
 
   return response.data;
 };
+
+export const getUserAccessToken = async () => {
+  const response = await APIClient.get<{
+    data: PersonalAccessToken[];
+  }>('/users/security/token');
+
+  return response.data.data;
+};
+
+export const createUserAccessTokenWithPut = async ({
+  JWTTokenExpiry,
+  tokenName,
+}: {
+  JWTTokenExpiry?: string;
+  tokenName?: string;
+}) => {
+  const response = await APIClient.put<
+    {
+      JWTTokenExpiry?: string;
+      tokenName?: string;
+    },
+    AxiosResponse<PersonalAccessToken[]>
+  >(`/users/security/token`, {
+    JWTTokenExpiry,
+    tokenName,
+  });
+
+  return response.data;
+};
+
+export const revokeAccessTokenWithPut = async (params: string) => {
+  const response = await APIClient.put<PersonalAccessToken>(
+    '/users/security/token/revoke?' + params
+  );
+
+  return response.data;
+};
```

Comment:
I don't think we need this

Reply:
We would need this as BE expects params in query parameters